dolphinscheduler-ui/src/views/projects/task/components/node/format-data.ts

```diff
@@ -409,7 +409,9 @@ export function formatParams(data: INodeData): {
       timeout: data.timeoutFlag ? data.timeout : 0,
       timeoutFlag: data.timeoutFlag ? 'OPEN' : 'CLOSE',
       timeoutNotifyStrategy: data.timeoutFlag ? timeoutNotifyStrategy : '',
-      workerGroup: data.workerGroup
+      workerGroup: data.workerGroup,
+      cpuQuota: data.cpuQuota,
+      memoryMax: data.memoryMax
```

Comment:
It is better to set a default value for the cupQuota and the memoryMax when other tasks don't need the two parameters.

Reply:
Done.